packages/webapp/src/components/InfoTooltip/InfoTooltip.tsx

```diff
@@ -8,12 +8,13 @@ import { memo, useCallback } from 'react'
 
 export interface InfoTooltipProps {
 	children?: JSX.Element
+	title?: string
```

Comment:
Should we add this in every place `InfoTooltip` is used? If so, we might also want to look for places where `TooltipWrapper` is used. I can help with that.

Reply:
Done! I updated instances of TooltipWrapper to use the label as the title and instances of InfoTooltip to have titles.